docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -373,6 +373,31 @@ To log out of the AWS application and remove credentials:
 $ tsh apps logout awsconsole-test
 ```
 
+## Troubleshooting
+### `remote error: tls: bad certificate` error during SSM sessions
+
+You may encounter the `remote error: tls: bad certificate` error when starting
+System Session Manager (SSM) sessions using `tsh aws ssm start-session` or `tsh
+aws ecs execute-command` commands.
+
+The issue is that `tsh` cannot properly proxy WebSocket connections sent by
+SSM.
+
+Please upgrade to the latest version of `tsh` where workarounds have been
+implemented for [`tsh aws ssm
+start-session`](https://github.com/gravitational/teleport/pull/30510) and [`tsh
+aws ecs
+execute-command`](https://github.com/gravitational/teleport/pull/33705).
```

Comment:
I would make it more explicit that the links are to PRs. Otherwise, a user could think they link to another docs page and ignore them. I would remove the links here and add the following at the end of this H3 section:

```markdown
For more information on the `tsh` workarounds, see the 
pull requests that introduced them:

- [https://github.com/gravitational/teleport/pull/30510](https://github.com/gravitational/teleport/pull/30510)
- [https://github.com/gravitational/teleport/pull/30510](https://github.com/gravitational/teleport/pull/33705)
```